acorn/index.d.ts

```diff
@@ -228,19 +218,23 @@ declare namespace acorn {
 
     const version: string;
 
-    function parse(input: string, options?: Options): ESTree.Program;
+    interface IParse {
+        (input: string, options?: Options): ESTree.Program;
+    }
+
+    const parse: IParse;
 
     function parseExpressionAt(input: string, pos: number, options?: Options): ESTree.Expression;
 
     function tokenizer(): Parser;
 
-    let parse_dammit: Function;
+    let parse_dammit: IParse;
```

Comment:
Even if keeping this way and not real function declaration, `const` would be better than `let`.